backend/src/main/resources/db/dev-data-migration/V0_0_2__initialData.sql

```diff
@@ -0,0 +1,13 @@
+TRUNCATE TABLE example CASCADE;
+
+
+INSERT INTO example (id, text)
+VALUES (1, 'Example 1'),
+       (11, 'Example 2'),
+       (21, 'Example 3'),
+       (31, 'Example 4'),
+       (41, 'Example 5');
```

Comment:
```suggestion
VALUES (1, 'Example 1'),
       (2, 'Example 2'),
       (3, 'Example 3'),
       (4, 'Example 4'),
       (5, 'Example 5');
```
I'd just change this to 1,2,3 ... and so on